package.json

```diff
@@ -87,7 +87,7 @@
     "testEnvironment": "node"
   },
   "peerDependencies": {
-    "@nestjs/common": "^8.0.0",
+    "@nestjs/common": "^8.0.0 || ^9.0.5",
```

Comment:
let's change it here to `^8.0.0 || ^9.0.0`

Reply:
sure